src/NServiceBus.Heartbeat.AcceptanceTests/ConfigureEndpointLearningPersistence.cs

```diff
@@ -24,8 +24,8 @@ public Task Configure(EndpointConfiguration configuration)
 
         storageDir = Path.Combine(tempDir, testRunId);
 
-        configuration.UsePersistence<InMemoryPersistence, StorageType.Subscriptions>();
-        configuration.UsePersistence<InMemoryPersistence, StorageType.Timeouts>();
+        configuration.UsePersistence<AcceptanceTestingPersistence, StorageType.Subscriptions>();
```

Comment:
You can delete this entire file since its been replaced by https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.AcceptanceTests/ConfigureEndpointAcceptanceTestingPersistence.cs